test/1.4/structure/T_name_parameter.dml

```diff
@@ -38,4 +36,4 @@ method test() throws {
     log info: "b_r_f qname is %s", b.r.f.qname;
     /// GREP .* c_r_f qname is c.NOT_THE_BANK.f
     log info: "c_r_f qname is %s", c.r.f.qname;
-}
\ No newline at end of file
+}
```

Comment:
here too a newline is added at the end of file

Reply:
I tend to add missing newlines at EOF. I don't have a particular strong aversion for them, only a weak one. Tools tend to agree with me, e.g. diff viewers marks missing nelwines as anomalies (it says "no newline at EOF" on them, rather than "newline at EOF" on my line)

I don't protest when people forget to add EOF newlines, but I don't mind adding them myself.